content/en/docs/ops/best-practices/traffic-management/index.md

```diff
@@ -156,7 +156,7 @@ in the client namespace.
 
 If you now send the request from a different namespace, for example `ns2`,
 the client is no longer in the same namespace as the destination rule, `ns1`.
-Because the corresponding service, `myservice.default.svc.cluster.local`, is also not in `ns1`,
+Because the corresponding service, `myservice.default.svc.cluster.local`, is also not in `ns2`,
```

Comment:
This change is not correct. The `DestinationRule` is in `ns1` in this example, so the request would work if the service was in also in `ns1` (i.e., the second namespace in the lookup path). If the service is in `ns2`, the `DestinationRule` (in `ns1`) will still not be found.

Reply:
My apologies Frank, thank you for the clarifications. I will close the pull request.